.github/workflows/release.yml

```diff
@@ -0,0 +1,84 @@
+name: Release and publish to npm
```

Comment:
As discussed earlier with you and @mhochsto we should consider using descriptors that can be concatenated to unique job names. Hence, it might make sense to call this "release".

Reply:
Ah right I did not check the names. We need to lint that somehow, so we don't have to do it anymore.